src/pages/console/uibuilder/override.mdx

```diff
@@ -47,6 +47,35 @@ import {FAQItem} from './ui-components'
 </tbody>
 </table>
 
+### Nesting Collections
```

Comment:
In retrospect, I think we should move this below the other section given that only in the next section, we explain "overrideItems"